build.sbt

```diff
@@ -301,11 +305,11 @@ lazy val python =
     )
     .dependsOn(core % "test->test")
 
-lazy val hail = (project in file("python/glow/hail"))
+lazy val hail = (project in file("hail"))
   .settings(
     pythonSettings,
     test in Test := {
-      hailtest.toTask(" --doctest-modules python/glow/hail/").value
+      hailtest.toTask(" --doctest-modules python/glow/hail/ docs/source/etl/hail.rst").value
```

Comment:
nit: make this into a variable `hailPaths` in case we have more hail related docs in the future